### PR TITLE
feat: add plugins crypto ✅

### PR DESCRIPTION
**Pull request by dajneem23**
👋 Hi there! I just made a plugin for Cypto price using [Coingecko API](https://www.coingecko.com/vi/api/documentation).
Please take a look at it

metrics_crypto

### CODE REVIEW

**Comment by dajneem23** [comment]
Screenshot 2023-09-19 at 16 26 57

Screenshot

**Comment by lowlighter** [comment]
Hi 👋 !

@dajneem23 It looks awesome !
I'll review it later this week 👍 (don't worry if the CI doesn't pass, this is fine 😅)

File: source/plugins/community/crypto/metadata.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+name: 🧩 Crypto
```
Comment (lowlighter) [inline source/plugins/community/crypto/metadata.yml:1]: Can you choose a different emoji as icon ? The puzzle one is used as a placeholder

File: source/plugins/community/crypto/index.mjs [hunk 2]
```diff
@@ -0,0 +1,73 @@
+//Setup
+export default async function({login, q, imports, data, account}, {enabled = false, extras = false} = {}) {
+  //Plugin execution
+  try {
+    //Check if plugin is enabled and requirements are met
+    if ((!q.crypto) || (!imports.metadata.plugins.crypto.enabled(enabled, {extras})))
+      return null
+
+        //Load inputs
+    let {id, days = 1, vs_currency = "usd", precision = 2 } = imports.metadata.plugins.crypto.inputs({data, account, q})
```
Comment (lowlighter) [inline source/plugins/community/crypto/index.mjs:10]: You can remove the default values here, are they're already handled by the parser of `metadata.yml` so it avoids confusion

**Review by lowlighter — CHANGES_REQUESTED** [review]
Just two minors changes requested but overall looks good to me 😄 !

File: source/plugins/community/crypto/metadata.yml (hunk 1 above)
Comment (dajneem23) [reply source/plugins/community/crypto/metadata.yml:1]: yes, i can

File: source/plugins/community/crypto/index.mjs (hunk 2 above)
Comment (dajneem23) [reply source/plugins/community/crypto/index.mjs:10]: ok

**Review by lowlighter — APPROVED** [review]

**Comment by lowlighter** [comment]
Thanks a lot for your contribution !

**Pull request merged**